cas_implementations/src/eav/file.rs

```diff
@@ -170,12 +169,12 @@ impl EntityAttributeValueStorage for EavFileStorage {
         &mut self,
         eav: &EntityAttributeValueIndex,
     ) -> Result<Option<EntityAttributeValueIndex>, HolochainError> {
-        let fetched = self.fetch_eavi(
-            Some(eav.entity()),
-            Some(eav.attribute()),
-            Some(eav.value()),
-            IndexQuery::default(),
-        )?;
+        let fetched = self.fetch_eavi(&EaviQuery::new(
+            Some(eav.entity()).into(),
+            Some(eav.attribute()).into(),
+            Some(eav.value()).into(),
+            IndexFilter::LatestByAttribute,
+        ))?;
```

Comment:
There are a ton of changes just like this in this PR (as many as there are `fetch_eavi` calls). So take comfort, dear reader, that you can skip anything like this pattern once you understand this one.

I added some `impl From<_> for EavFilter` to make these minimally invasive.